src/status_im/node/core.cljs

```diff
@@ -124,18 +124,20 @@
                       (get-node-config db network))
         node-config-json (types/clj->json node-config)]
     (log/info "Node config: " node-config-json)
-    {:db         (assoc db :network network)
+    {:db         (assoc db :network           network
+                        :status-node-busy? true)
```

Comment:
could we keep the semantics of signals ? If I understand correctly we receive `node.ready` when the node is done starting ? So it would be better to us `status-node-started?` in the code when the node has been started and `status-node-ready?` when the node is now ready instead of busy? while it is started.
wdyt ?

Reply:
one thing here is that `status-node-started?` in that case won't match `node.started` signal, because we want to set one of the booleans to be a guard showing that the node is int the process of starting (set it to `true` with calling `start-node`).
`node.started` from status-go comes much later.

maybe the best option is to have a keyword enum `:stopped` / `:busy` / `:started` and keep one variable...

Reply:
I'm ok with an enum, having a state machine is less error prone and gives a clear understanding of what transitions are acceptable or not.

As I understand we have `:stopped` `:starting` (I prefer that over `:busy`) `:started`
From there it will be easier to determine which transition are acceptable or not

Reply:
okay, so it can't be `:stopping`, right? because that's why I chose `:busy` because I don't care if it is stopping or starting. but if `stopNode` is instantaneous, then `:starting` it's ok.

Reply:
done